### DISCUSSION TITLE
[Enhancement] [date-picker] 希望日期组件能单独导出日期面板，不需要输入框（I hope the date component can export the date panel separately without requiring an input box.）

### DISCUSSION THREAD

**Discussion by webpig** [element-plus/element-plus#12408] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `date-picker`

## Description

平时业务开发中，UI 设计给的样式和 el-date-picker 样式不一致；而且触发方式也可能不一样；如果可以使用单独的面板就很方便了。（In regular business development, the UI design provided may not be consistent with the style of el-date-picker, and the triggering method may also be different. It would be very convenient if a separate panel could be used.）

### Additional comments

_(empty)_

**Comment by webpig** [comment]
el-date-picker 没有 align 属性了

**Comment by vela666** [comment]
大佬你解决这个问题了吗

**Reply by webpig** [reply]
有的通过 CSS 样式来做了；有的自己实现了

**Reply by vela666** [reply]
好吧

**Reply by webpig** [reply]
没法子了，之前用 Vue2 的时候还能引入，路径通过 webpack 来转下。不过也不是很方便。

**Reply by vela666** [reply]
![image](https://github.com/element-plus/element-plus/assets/69613314/b9ef342e-cc95-4ee1-8287-8001df7a1598)
想自定义成这样 完全没一点办法现在...

**Reply by webpig** [reply]
只能自己实现了；你这个 UI 得有个面板放那。除非是让它可展开的倒还可以用下组件。

**Reply by webpig** [reply]
或者你试试看能不能让它一直展开

**Reply by webpig** [reply]
快捷键功能自己写，
![image](https://github.com/element-plus/element-plus/assets/24989439/348579df-1b31-4979-a36a-261da218f877)
这块的输入框隐藏，贴自己的样式上去。

**Reply by vela666** [reply]
https://github.com/mengxiong10/vue-datepicker-next/issues/48   我用这个插件 实现了

**Reply by webpig** [reply]
哦哦，你引入了另外的插件，这个之前好像看到过；

**Comment by Dsaquel** [comment]
Released in `2.11.0`.
https://element-plus.org/en-US/component/date-picker-panel.html
